src/elm/type.coffee

```diff
@@ -50,7 +50,12 @@ module.exports.ToDate = class ToDate extends Expression
 
   exec: (ctx) ->
     arg = @execArgs(ctx)
-    if arg? and typeof arg != 'undefined' then Date.parse(arg.toString()) else null
+    if (not arg?) or (arg == undefined)
```

Comment:
This is fine, but for future reference, I think `not arg?` will also be true for `undefined` -- so you don't need that second `arg == undefined`.